packages/dialtone-vue2/recipes/conversation_view/editor/editor.vue

```diff
@@ -102,7 +102,7 @@
               :input-aria-label="showAddLink.setLinkInputAriaLabel"
               data-qa="dt-editor-link-input"
               :placeholder="setLinkPlaceholder"
-              input-wrapper-class="d-bgc-black-100 d-mt6 d-bar5 d-ba d-baw1 d-bc-black-300 d-py2 d-ol-none"
+              input-wrapper-class="d-bgc-secondary d-mt6 d-bar5 d-ba d-baw1 d-bc-default d-py2 d-ol-none"
```

Comment:
We shouldn't be using utility classes here as it's more difficult to override if needed, but that was not really introduced by you so this should be fine.

Created a ticket so we can fix that later.